packages/showcase/src/app/components/drop-down/metadata.ts

```diff
@@ -0,0 +1,138 @@
+import { ComponentMetadata } from "../../../common/types";
+import { chameleonImportType } from "../../../services/template-language/create-template";
+import {
+  nestedModel,
+  plainModel,
+  withSeparatorsModel,
+  withSlotModel,
+  withIconsModelForMetadata
+} from "./models";
+
+// Common
+const DROP_DOWN_TAG = "ch-action-menu-render";
+const DROP_DOWN_CLASS = "dropdown";
+const CHILDREN = "Open Menu";
+const PROPERTIES = [
+  { name: "model", value: "actionMenuModel", state: true },
+  { name: "expanded", value: false, variable: true },
+  { name: "inlineAlign", value: "inside-start" },
+  { name: "blockAlign", value: "outside-end" }
+];
+const IMPORTS_NO_ICONS = [chameleonImportType("ActionMenuModel")];
+
+export const dropDownMetadata = {
```

Comment:
We should add more advanced cases, see:
https://www.figma.com/design/O05P2SkMK7aydI2Lxyu7Wi/Mercury-Components?node-id=3062-6779&t=ie3UcELDmZ5ryUmV-0

Also, we should add cases where the alignment of the dropdown changes.